FlowCrypt/Functionality/DataManager/Encrypted Storage/EncryptedStorage.swift

```diff
@@ -62,7 +65,7 @@ final class EncryptedStorage: EncryptedStorageType {
     private lazy var migrationLogger = Logger.nested(in: Self.self, with: .migration)
     private lazy var logger = Logger.nested(Self.self)
 
-    private let currentSchema: EncryptedStorageSchema = .version7
+    private let currentSchema: EncryptedStorageSchema = .version8
```

Comment:
I removed `user` property of `ClientConfigurationRealmObject` (https://github.com/FlowCrypt/flowcrypt-ios/blob/bugfix/issue-1467-lazy-fes-url/FlowCrypt/Models/Realm%20Models/ClientConfigurationRealmObject.swift), as it wasn't used anywhere. We use only `userEmail` property.

This change made possible using only email instead of full `User` object for storing client configuration - https://github.com/FlowCrypt/flowcrypt-ios/pull/1480/files#diff-ec0dcddd1520d18166856405ff33ab1276d052b7c05ef5148bdfe6100d8f640dR48

Reply:
Sounds good :+1: